.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Related Issue
+*Please link to the issue this pull request addresses. If there isn't one yet, please file one
+prior to creating the pull request so that it can be discussed.*
+
+## Testing
+*Please describe what you have done to test your changes.*
+
+## Checklist
+- [ ] I have read the [Arcade Contributor Guidance doc](https://github.com/dotnet/arcade/blob/master/Documentation/Policy/ArcadeContributorGuidance.md)
```

Comment:
This looks very heavy-weight to be filling out on every PR.

Reply:
I think that is a fair criticism and it's fine to dial it back. Mostly going off of what I've seen from other open source projects, but obviously we need to find something that works for us. This one is heavier weight because it will be used by outside contributors, but I understand the practical implications for day-to-day work.